README.md

```diff
@@ -112,6 +112,29 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS isnatllataion and compile simplewallet from sources. Below is a sample procedure for python TP/client:
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
+2. Clone sawtooth-simplewallet repo
+3. `cd sawtooth-simplewallet`
+4. Setup the simplewallet-tp.
+ - Follow the [`pyprocessor/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile#L18) in the Dockerfile
+ - Add the `pyprocessor` directory to your `PATH` environment, as shown below:
+
+`export PATH = $PATH:<absolute-path-to-your-cloned-sawtooth-simplewallet-dir>/pyprocessor"`
+ - Run the simplewallet-tp by following command:
+
+   `simplewallet-tp`
+5. Setup the client. Open a new console.
+ - Follow the [`pyclient/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyclient/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyclient/Dockerfile#L20) in the pyclient/Dockerfile
+ - Add the `pyclient` directory to your `PATH` environment, as shown below:
+
+`export PATH = $PATH:<absolute-path-to-your-cloned-sawtooth-simplewallet-dir>/pyclient"`
+ - Run the client by calling `simplewallet` command. Refer Usage section above.
+
```

Comment:
I would remove the step for adding the `pyclient` directory to `PATH` (lines 133-136) and replace with this instruction instead:

> Start the simplewallet client with `./pyclient/simplewallet`